scrapers/base.py

```diff
@@ -13,8 +13,9 @@ class BaseScraper(object):
 
     teiid_host = 'virtualdb.engineering.redhat.com'
     teiid_port = 5432
-    # Default start date to fetch data from
+    # Default start date and end date to fetch data
     default_since = datetime(2016, 3, 1)
```

Comment:
Optional: I know you didn't set this in this PR but it'd be nice to default to 365 days ago instead of a static date.